cli/setup.py

```diff
@@ -46,6 +46,7 @@ def read(fname):
     zip_safe = False,
     package_data = {
         '' : ['examples/config'],
+        'stepfunctions': ['*']
```

Comment:
Does this include all files/directories below setup.py?

Reply:
it includes all of the files/directories below stepfunctions/

Reply:
Figured out that it doesn't do it recursively (the src/ and templates/ folders are not added) so I changed this to 'stepfunctions': ['**/*']